pkg/storage/snapshot/snapshot.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strings"
 	"time"
 
+	metrics "kubevirt.io/kubevirt/pkg/monitoring/metrics/virt-controller"
```

Comment:
I think this should be moved next to the "kubevirt.io/kubevirt/pkg/controller" import

Reply:
fixed